packages/headless/DocJsonParser.rb

```diff
@@ -0,0 +1,438 @@
+require 'json'
```

Comment:
I would like this file to be javascript at a minimum (typescript could also be nice, but can come later). The reason is using ruby to process the docs adds significant complexity to the ui-kit project, for not much return.

Having a second language means a more complex local-setup, more complex docker images in jenkins CI, needing contributors to learn ruby syntax, and needing to install ruby-dependencies if we want to write unit tests for the code.